package.json

```diff
@@ -50,6 +50,58 @@
       "ecmaVersion": 2019
     }
   },
+  "husky": {
+    "hooks": {
+      "commit-msg": "commitlint -E HUSKY_GIT_PARAMS"
+    }
+  },
+  "commitlint": {
+    "rules": {
+      "body-leading-blank": [
+        2,
+        "always"
+      ],
+      "body-max-line-length": [
```

Comment:
maybe add `subject-max-length` for good measure? its hidden in the docs for `commitlint/config-conventional`, not in the readme

Reply:
Thanks!  I had it in there, looks like I killed it by accident.  Good catch.